tests/utils.py

```diff
@@ -1,13 +1,20 @@
 import contextlib
 import http.server
+import os
 import pathlib
 import threading
 from ssl import PROTOCOL_TLS_SERVER, SSLContext
 
+import filelock
```

Comment:
This one should be marked as an external dependency

Reply:
Can you help me how to do that, please?